pkg/operator/sync.go

```diff
@@ -423,6 +423,7 @@ func newContainers(config *OperatorConfig, features map[string]bool) []corev1.Co
 		"--logtostderr=true",
 		"--v=3",
 		"--leader-elect=true",
+		"-leader-elect-lease-duration=90s",
```

Comment:
why only one dash here while any other command has `--`?

Reply:
Copy pasting from CLI is an unhealthy habit 😅Even when it works both ways, will fix in a second